docs/dev/feature-usage.md

```diff
@@ -0,0 +1,45 @@
+## Feature Usage
+
+Feature usage is a JSON-formatted field on the Cluster object designed for monitoring  
+how often a feature of Assisted Service is being used.
+
+Developers should add an entry to the feature usage structure for each customer-facing feature.  
+The data can be as simple as marking whether a feature is configured or not (the default setup)  
+or more elaborate, depending on the data the Product owners like to get.
+
+For example, we report which network type is employed (OVN, SDN) to monitor Network type configuration. 
+For monitoring usage of Additional NTP Sources, we also add a property that counts how many sources  
+the customer added.
+
+### How to add a usage entry
+
+1. Add a constant to [usage/consts.go](https://github.com/openshift/assisted-service/blob/master/internal/usage/consts.go)
+2. If the feature behavior is set at cluster creation time, add a call to SetUsage in [setDefaultUsage](https://github.com/openshift/assisted-service/blob/master/internal/bminventory/inventory.go#L3236)
```

Comment:
lines can change, you need to point to a specific commit, for example https://github.com/openshift/assisted-service/blob/52fe780cb49242dc1bddc2fcba490a9047ee17fa/internal/bminventory/inventory.go#L3236

Reply:
same comment to the other lines references, when you in the github page press `y` it will switch the url to commit url.